website/versioned_docs/version-2.0.0-alpha.48/styling-layout.md

```diff
@@ -131,3 +131,75 @@ The class names which will be processed by webpack into a globally unique class
 _This section is a work in progress._ [Welcoming PRs](https://github.com/facebook/docusaurus/issues/1640).\_
 
 :::
+
+### Sass/SCSS
```

Comment:
Do not make new changes to any versioned docs. Please move this section to `website/docs/styling-layout.md`.

Reply:
Makes sense. I just clicked the `Edit this page` buttons without thinking too much about it. 
I updated the PR.